xrdp/xrdp_mm.c

```diff
@@ -1218,29 +1247,26 @@ xrdp_mm_egfx_caps_advertise(void *user, int caps_count,
                 best_pro_index = index;
                 break;
             case XR_RDPGFX_CAPVERSION_10:
-                best_pro_index = index;
-                break;
-            case XR_RDPGFX_CAPVERSION_101:
-                best_pro_index = index;
-                break;
-            case XR_RDPGFX_CAPVERSION_102:
-                best_pro_index = index;
-                break;
-            case XR_RDPGFX_CAPVERSION_103:
-                best_pro_index = index;
-                break;
-            case XR_RDPGFX_CAPVERSION_104:
                 if (!(flags & XR_RDPGFX_CAPS_FLAG_AVC_DISABLED))
                 {
                     best_h264_index = index;
                 }
                 best_pro_index = index;
                 break;
-            case XR_RDPGFX_CAPVERSION_105:
+            case XR_RDPGFX_CAPVERSION_102: /* FALLTHROUGH */
+            case XR_RDPGFX_CAPVERSION_103: /* FALLTHROUGH */
+            case XR_RDPGFX_CAPVERSION_104: /* FALLTHROUGH */
+            case XR_RDPGFX_CAPVERSION_105: /* FALLTHROUGH */
+            case XR_RDPGFX_CAPVERSION_106: /* FALLTHROUGH */
+            case XR_RDPGFX_CAPVERSION_107:
+                if (!(flags & XR_RDPGFX_CAPS_FLAG_AVC_DISABLED))
+                {
+                    best_h264_index = index;
+                }
                 best_pro_index = index;
                 break;
-            case XR_RDPGFX_CAPVERSION_106:
-                best_pro_index = index;
+            default:
+                /* just skip unknwown */
```

Comment:
Add a log statement that we found an unknown cap, probably this should be LOG(LOG_INFO...)